pkgs/development/python-modules/manim-voiceover/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, pytest-xdist
+, pytestCheckHook
+, manim
+, mutagen
+, pydub
+, humanhash3
+, sox
+, pip
+, python-dotenv
+, gtts
+, pynput
+, pyaudio
+, playsound
+, stable-ts
+, pygobject3
+, gst_all_1
+, wrapGAppsHook
+, intltool
+}:
+
+buildPythonPackage rec {
+  pname = "manim-voiceover";
+  format = "pyproject";
+  version = "0.3.0";
+  disabled = pythonOlder "3.7";
+
+  extras = [ "gtts" "recorder" ];
+
+  src = fetchFromGitHub {
+    owner  = "ManimCommunity";
+    repo = "manim-voiceover";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-ugR435frhpleyzHG6cbKnTTyqRVpRKPBLkaZQ+E8Cwo=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    wrapGAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'sox = "^1.4.1"' 'sox = "*"'
+  '';
+
+  propagatedBuildInputs = [
+    manim
+    mutagen
+    pydub
+    humanhash3
+    sox
+    python-dotenv
+    pip
```

Comment:
We really need pip?

Reply:
I removed it and it still build, so I assume no.
I just took what was in the dependency file of the project.

Reply:
I put it back because it build but doesn't run properly.